modular_zubbers/code/datums/elements/examine_when_worn.dm

```diff
@@ -0,0 +1,16 @@
+// This is cheaper than adding to the Topic() of atom
+/datum/element/examined_when_worn
+
+/datum/element/examined_when_worn/Attach(datum/target)
+	. = ..()
+	RegisterSignal(target, COMSIG_TOPIC, PROC_REF(on_topic))
+	ADD_TRAIT(target, TRAIT_WORN_EXAMINE, TRAIT_SUBTREE_REQUIRED_OPERATIONAL_DATUM)
```

Comment:
I don't see where you're removing the trait

Reply:
Oh true 💀